makefile

```diff
@@ -31,11 +33,18 @@ run-sim2play: all
 	cd run; ./simulator --headless &
 	cd run; ./soccer -sim -y & ./soccer -sim -b
 
+debug: all
+	$(GDB) ./run/soccer
+
+debug-sim: all
+	-pkill -f './simulator --headless'
+	cd run; ./simulator --headless &
+	gdb ./run/soccer
```

Comment:
Also, if you do `gdb --args run/soccer -sim`, you won't have to type `-sim` after gdb starts.